packages/discord.js/src/structures/ThreadChannel.js

```diff
@@ -293,15 +293,11 @@ class ThreadChannel extends BaseChannel {
    * @param {BaseFetchOptions} [options] The options for fetching the member
    * @returns {Promise<?ThreadMember>}
    */
-  async fetchOwner({ cache = true, force = false } = {}) {
-    if (!force) {
-      const existing = this.members.cache.get(this.ownerId);
-      if (existing) return existing;
-    }
+  async fetchOwner(options) {
+    if (!this.ownerId) return null;
 
-    // We cannot fetch a single thread member, as of this commit's date, Discord API responds with 405
-    const members = await this.members.fetch({ cache });
-    return members.get(this.ownerId) ?? null;
+    const member = await this.members.fetch({ ...options, member: this.ownerId }).catch(() => null);
+    return member;
```

Comment:
```suggestion
    return await this.members.fetch({ ...options, member: this.ownerId }).catch(() => null);
 ```
 
Also is this catch desired here?

Reply:
Yes it is, so the method returns null in case the owner of the thread left the guild.

Reply:
Whilst correct, I disagree this should be present. Silently catching errors and doing nothing here is not a good solution. For example, this removes the possibility of returning abortion errors.

There is no documentation to inform the condition when this returns `null`, so perhaps we could use that to our advantage. Line 297 could stay and `.catch(() => null)` could be removed. Then, we document this behaviour.